Update single-and-multi-tenant-apps.md (#1343)
MicrosoftDocs/entra-docs: docs/identity-platform/single-and-multi-tenant-apps.md

- Follow the principle of least user access to ensure that your app only requests permissions it actually needs.
- Provide appropriate names and descriptions for any permissions you expose as part of your app. This helps users and admins know what they're agreeing to when they attempt to use your app's APIs. For more information, see the best practices section in the [permissions guide](./permissions-consent-overview.md).

> [!NOTE]

> > Multi-tenant applications can be deployed to the same National cloud instances, but not acrossed Nation Cloud instances.

Reviewer: ```suggestion
> Multitenant applications can be deployed to the same national cloud instances, but not across national cloud instances.
```
making requested change

> > [Azure National Clouds](https://learn.microsoft.com/en-us/entra/identity-platform/authentication-national-cloud)

Reviewer 2: We try to use relative links when possible.  -> [Azure National Clouds](/entra/identity-platform/authentication-national-cloud)

> > [Azure National Clouds](https://learn.microsoft.com/en-us/entra/identity-platform/authentication-national-cloud)

Reviewer: ```suggestion
> [Azure National Clouds](/authentication-national-cloud.md)
```
making requested change

> Examples:

> > - A Multi-tenant application created in a commercial tenant, can be added to other commercial tenants.

Reviewer 2: Minor edits:  A multitenant application created in a commercial tenant can be added to other commercial tenants.

> > - A Multi-tenant application created in a commercial tenant, can be added to other commercial tenants.

Reviewer: ```suggestion
> - A multitenant application created in a commercial tenant can be added to other commercial tenants.
```

> > - A Multi-tenant application created in an Azure Government tenant, can be added to other Azure Government tenants.

Reviewer 2: Minor edits:  A multi-tenant application created in an Azure Government tenant can be added to other Azure Government tenants.

> > - A Multi-tenant application created in an Azure Government tenant, can be added to other Azure Government tenants.

Reviewer: ```suggestion
> - A multi-tenant application created in an Azure Government tenant can be added to other Azure Government tenants.
```
